monitoring/sensu_check.py

```diff
@@ -320,6 +356,7 @@ def main():
                 'aggregate':    {'type': 'bool'},
                 'low_flap_threshold':  {'type': 'int'},
                 'high_flap_threshold': {'type': 'int'},
+                'custom':   {'type': 'str'},
```

Comment:
You can just use `dict` as the type here. In fact, that would make the code a lot simpler. Ansible will then make sure that your param is properly formatted :-)